packages/uswds-utilities/src/styles/rules/border-color.scss

```diff
@@ -1,16 +1,16 @@
 /*
 ========================================
-background-color
+border-color
 ----------------------------------------
 usage:
-  .border-[color]
+  .border-[key]
 ----------------------------------------
 output:
-  border[-color]: [color];
+  border-color: [value];
 ----------------------------------------
 example:
   .border-red {
-    border-color: #be4900; }
+    border-color: #e52207; }
```

Comment:
```suggestion
    border-color: #e52207; 
  }
```

Reply:
This was updated in [267fc6d](https://github.com/uswds/uswds/pull/5859/commits/267fc6d3e58a1be2aaff128aeca4f8e6fad024b8)